### PR TITLE
Resolved some CS1573,1591

### PR DESCRIPTION
**Pull request by WaldemarCoding**
﻿## Description of Change

Resolved some CS1573,1591 #1978 

### Bugs Fixed

- Resolved some CS1573,1591 #1978 

### API Changes

None

### Behavioral Changes

None 

### PR Checklist

- [ ] Has tests (if omitted, state reason in description)
- [X] Rebased on top of master at time of PR
- [X] Changes adhere to coding standard

### CODE REVIEW

File: src/Forms/Prism.Forms/Navigation/INavigationParameters.cs [hunk 1]
```diff
@@ -2,20 +2,57 @@
...
+        /// <param name="key">The key for the value to be returned</param>
+        /// <returns>Returns a matching parameter of <see cref="T" /> if one exists in the Collection</returns>
         T GetValue<T>(string key);
 
+        /// <summary>
+        /// Returns an IEnumerable of all parameters 
+        /// </summary>
+        /// <typeparam name="T">The type for the values to be returned</typeparam>
+        /// <param name="key">The key for the values to be returned</param>
+        ///<returns>Returns a IEnumberable of all the instances of type <see cref="T"/></returns>
```
Comment (d3fkn1ght) [inline src/Forms/Prism.Forms/Navigation/INavigationParameters.cs:47]: I've noticed that the use of <see cref="T"/> introduces another compiler warning.  I've been using <paramref name="T"/> instead.

File: src/Forms/Prism.Forms/Navigation/INavigationParameters.cs [hunk 2]
```diff
@@ -2,20 +2,57 @@
 
 namespace Prism.Navigation
 {
+    /// <summary>
+    /// Provides a way for the INavigationService to pass parameteres during navigation.
```
Comment (dansiegel) [inline src/Forms/Prism.Forms/Navigation/INavigationParameters.cs:6]: INavigationService -> `<see cref="INavigationService" />`

Comment (dansiegel) [reply src/Forms/Prism.Forms/Navigation/INavigationParameters.cs:6]: There is no need for docs on anything in the body as we're removing the contents to a common IParameters

File: src/Wpf/Prism.Wpf/Bootstrapper.cs [hunk 3]
```diff
@@ -21,7 +21,12 @@ namespace Prism
     [Obsolete("It is recommended to use the new PrismApplication as the app's base class. This will require updating the App.xaml and App.xaml.cs files.")]
     public abstract class Bootstrapper
     {
+        /// <summary>
+        /// Gets the <see cref="IContainerExtension"/> for the application.
+        /// </summary>
+        /// <value>A <see cref="IContainerExtension"/> instance.</value>
```
Comment (dansiegel) [inline src/Wpf/Prism.Wpf/Bootstrapper.cs:27]: make sure we rebase... I know there were changes to this file

**Comment by WaldemarCoding** [comment]
I think I messed up the rebase here. Not sure what happend here.
Took my master and rebased after Fetching upstream/master. 
Why are @brianlagunas changes are now part of this PR?

File: src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs [hunk 4]
```diff
@@ -1,11 +1,30 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Internal - Provides a way for the <see cref="INavigationService"/> to pass parameteres during navigation.
+    /// </summary>
     public interface INavigationParametersInternal
     {
+        /// <summary>
+        /// Adds the key and value to the KeyValuePair<string,object> collection
```
Comment (dansiegel) [inline src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs:9]: this is invalid markup... second we don't really want to reference the KeyValuePair's outright... just refer to them as parameters

File: src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs [hunk 5]
```diff
@@ -1,11 +1,30 @@
 ﻿namespace Prism.Navigation
 {
+    /// <summary>
+    /// Internal - Provides a way for the <see cref="INavigationService"/> to pass parameteres during navigation.
```
Comment (dansiegel) [inline src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs:4]: Used to set internal parameters used by Prism's NavigationService

File: src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs [hunk 6]
```diff
@@ -1,11 +1,30 @@
...
+    /// </summary>
     public interface INavigationParametersInternal
     {
+        /// <summary>
+        /// Adds the key and value to the KeyValuePair<string,object> collection
+        /// </summary>
+        /// <param name="key">The key to reference this value in the KeyValuePair<string, object></param>
```
Comment (dansiegel) [inline src/Forms/Prism.Forms/Navigation/INavigationParametersInternal.cs:11]: see comment above

**Pull request merged**